src/index.js

```diff
@@ -0,0 +1,59 @@
+import './style.css';
+import reload from './assets/reload.png';
+import move from './assets/vertical-dots.png';
+
+const tasks = [
+  {
+    description: 'task 1',
+    done: false,
+    id: 1,
+  },
+  {
+    description: 'task 2',
+    done: true,
+    id: 1,
+  },
+  {
+    description: 'task 3',
+    done: false,
+    id: 1,
+  },
+];
```

Comment:
- Though your app is working fine. But there is a bug I can see. There are three items but every item has the same `id`. 
- In the project requirements we can see that: "On page load render the dynamically created list of tasks in the dedicated placeholder. `The list should appear in order of the index values for each task.`"
- `The list should appear in order of the index values for each task` this means `id` should be different for each task.
- [ ] Kindly fix the `id` of every item.